src/core_atmosphere/physics/Makefile

```diff
@@ -248,11 +248,6 @@ mpas_atmphys_rrtmg_swinit.o: \
 	mpas_atmphys_constants.o \
 	mpas_atmphys_utilities.o
 
-mpas_atmphys_driver_seaice.o: \
-	mpas_atmphys_constants.o \
-	mpas_atmphys_lsm_shared.o \
-	mpas_atmphys_vars.o
-
```

Comment:
I take it this was removed/moved to the new core_seaice. Not sure if a whole new core is necessary, but I definitely wasn't invited to the design plan meeting.